edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_navigation.scss

```diff
@@ -1,175 +1,179 @@
-// LMS -- elements -- navigation
-// ====================
-
-// in this document:
-// --------------------
-// +notes
-// +skip navigation
-// +utility navigation
-// +toggling utilities
-// +case - calculator spacing
-// +tabs
-
-// +notes:
-// --------------------
-// this Sass partial should have its contents eventually abstracted out so that onboarding/non-coureware navigation is separate from in course-based navigation systems
-
-
-// +skip navigation
-// --------------------
-%nav-skip {
-  @extend %text-sr;
+.global-header h1.header-logo a .logo {
+    float: left;
+    height: 55px;
+    margin-left: 3px;
 }
 
-.nav-contents,
-.nav-skip,
-.transcript-skip {
-  @extend %nav-skip;
+.global-header {
+  min-height: 70px;
+  position: relative;
+  z-index: 1;
+  overflow: inherit;
 }
 
-// +utility navigation (course utiltiies)
-// --------------------
-.nav-utilities {
-  @extend %ui-depth3;
+.vue-header.global-header {
+  background-color: $vue-header-footer !important;
+  border: none;
+  color: $vue-btn-text;
+
+      @media (min-width: 992px) {
+          .tab-nav-link {
+              color: $vue-btn-text !important;
+           }
+      }
+
+      .nav-links .secondary .nav-item a {
+        color: $vue-btn-text;
+      }
+
+      .nav-links .secondary .dropdown-user-menu .dropdown-item a {
+         color: $vue-dark ;
+      }
+
+      .header-logo .course-header {
+         color: $white;
+
+        .course-name {
+          display: block;
+          font-size: $font-size-base;
+          margin-top: 15px !important;
+          font-weight: 600;
+        }
+      }
+}
 
-  position: fixed;
-  right: ($baseline/4);
-  bottom: 0;
+.apprenticeships-header {
+  background-color: $app-header;
+ // border: none;
+  color: $app-btn-text;
 
-  .wrapper-utility {
-    @extend %wipe-last-child;
+  @media (min-width: 992px) {
+    .tab-nav-link {
+      color: $app-btn !important;
+    }
+  }
 
-    display: inline-block;
-    vertical-align: bottom;
+   .nav-links .secondary .nav-item a, .fa {
+      color: $app-header-text
+    }
 
-    @include margin-right(6px);
+    .nav-links .secondary .dropdown-user-menu .dropdown-item a {
+     color: $app-header-text;
   }
+}
 
-  .utility-control {
-    @include transition(background-color $tmg-f2 ease-in-out 0s, color $tmg-f2 ease-in-out 0s);
+.vue-btn:hover {
+  background-color: $vue-btn-text !important;
+  border: 1px solid $vue-btn !important;
+  color: $vue-btn !important;
+}
 
-    position: relative;
-    bottom: -($baseline/5);
-    display: inline-block;
-    vertical-align: middle;
-    padding: ($baseline/2) ($baseline*0.75) ($baseline*0.75) ($baseline*0.75);
-    background: $black-t1;
-    color: $white;
-
-    // STATE: hover/active
-    &:hover,
-    &:active {
-      background: $gray-d1;
-    }
+.apprencticeships-btn:hover {
+  background-color: $app-btn-text !important;
+  border: 1px solid $app-btn !important;
+  color: $app-btn  !important;
```

Comment:
Remove the additional space before the "!important" definition. @luismorenolopera